src/calculateRentalCost.js

```diff
@@ -20,7 +20,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return (days * 40);
+  } else if (days < 7) {
+    return (days * 40) - 20;
+  } else {
+    return (days * 40) - 50;
```

Comment:
```suggestion

    return (days * 40) - 50;
```

Reply:
must be whitespace before `return`
https://github.com/mate-academy/style-guides/blob/master/javascript.md#whitespace--after-blocks